install/installer/cluster_dev.py

```diff
@@ -237,17 +330,56 @@ def parse_cli_args():
         help='Automatically initialize repo for you',
     )
     parser.add_argument(
-        '--git-user-name', metavar='<username>',
+        '--git-user-name', '-gusr', metavar='<username>',
```

Comment:
Having short args longer than one char is uncommon. Do you really need this?

Reply:
Maybe no, will be removed